packages/javascript-sdk/src/token-manager/helpers.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * @forgerock/javascript-sdk
+ *
+ * helpers.test.ts
+ *
+ * Copyright (c) 2022 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+import { tokensWillExpireWithinThreshold } from './helpers';
+import {
+  oauthThreshold,
+  tokenExpiryWithinThreshold,
+  tokenExpiryOutsideThreshold,
+} from './token-manager.mock.data';
+
+describe('Test Token Manager utils', () => {
+  it('Should return true if tokens expire within configured threshold', async () => {
+    const expiresSoon = tokensWillExpireWithinThreshold(
+      oauthThreshold,
+      tokenExpiryWithinThreshold,
+      Date.now(),
```

Comment:
While I don't foresee a situation where using the actual Date.now() would be problematic in this scenario, I feel somewhat uneasy about using Date.now's actual value in test suite given its ever changing. 

Typically, using IO in unit tests requires mocking, and while this may be a scenario where we can get away with it, my brain has a hesitation about this. What do you think?

Reply:
Yes I agree it doesn't quite feel right doing it like this but was kind of on the fence. Would the alternative be to put a static timestamp in the mock data and use that for the tests?

Reply:
I've removed the date from here, feels more right. Thanks